fleet/api/openapi.yaml

```diff
@@ -267,6 +421,55 @@ components:
           description: Maximum number of items to return in one page.
       required:
         - agent_groups
+    AgentUpdateReqSchema:
+      type: object
+      properties:
+        name:
+          type: string
+          description: A unique name label
+          example: eu-agents
+        agent_tags:
```

Comment:
`agent_tags` is the one that is read only from the API, and set from the agent itself. So this should be `orb_tags` instead

Reply:
Done